src/Socket/groups.ts

```diff
@@ -348,7 +350,8 @@ export const extractGroupMetadata = (result: BinaryNode) => {
 		participants: getBinaryNodeChildren(group, 'participant').map(({ attrs }) => {
 			return {
 				id: attrs.jid,
-				jid: attrs.phone_number ? jidNormalizedUser(attrs.phone_number) : undefined,
+				jid: isJidUser(attrs.jid) ? attrs.jid : jidNormalizedUser(attrs.phone_number),
+				lid: isLidUser(attrs.jid) ? attrs.jid : attrs.lid,
```

Comment:
Is this really correct? Shouldn't it be:
```suggestion
				lid: isLidUser(attrs.jid) ? attrs.lid : attrs.jid,
```